convmv.md

```diff
@@ -0,0 +1,7 @@
+# convmv
+
+> Converts filenames from one encoding to another
+
+- convert encoding without dry run
```

Comment:
😎
1 янв. 2016 г. 21:23 пользователь "Waldir Pimenta" <notifications@github.com>
написал:

> In convmv.md
> <https://github.com/tldr-pages/tldr/pull/531#discussion_r48679763>:
>
> > @@ -0,0 +1,7 @@
> > +# convmv
> > +
> > +> Converts filenames from one encoding to another
> > +
> > +- convert encoding without dry run
>
> "without dry" run is confusing. The default invocation does a dry run in
> addition to the conversion, or instead of it? Maybe add an example above
> this one without the --notest, to better clarify the command's behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tldr-pages/tldr/pull/531/files#r48679763>.
>